controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -27,6 +27,7 @@ import (
 const (
 	KubeadmControlPlaneFinalizer    = "kubeadm.controlplane.cluster.x-k8s.io"
 	KubeadmControlPlaneHashLabelKey = "kubeadm.controlplane.cluster.x-k8s.io/hash"
+	SelectedForUpgradeLabel         = "kubeadm.controlplane.cluster.x-k8s.io/selected-for-upgrade"
```

Comment:
can you update the docs as well to include this label? I assume humans can add this and then select a particular node for upgrade. If that's a false assumption and this is internal only, then this shouldn't be exported and we don't need docs for it